src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -202,6 +202,7 @@
 
     <MakeDir Directories="$(VisualStudioBuildPackagesDir)"/>
     <WriteLinesToFile Lines="@(_Dependency->'%(_AssemblyName),%(_AssemblyVersion)')" File="$(_DependentAssemblyVersionsFile)" Overwrite="true"/>
+    <WriteLinesToFile Lines="Roslyn,$(AssemblyVersion)" File="$(_DependentAssemblyVersionsFile)" Overwrite="false" />
```

Comment:
`DependsOnTargets` needs to list `GetAssemblyVersion` target to ensure `AssemblyVersion` is initialized.